gradle.properties

```diff
@@ -0,0 +1 @@
+releaseVersion=0.1
```

Comment:
should we add a milestone version here? so `0.1.M1`? what do you think?

Reply:
Makes sense in my opinion.